index.html

```diff
@@ -56,6 +56,35 @@
     SHOULD join the mailing lists below and take part in the discussions.</p>
   </section>
   <section>
+      <h2>
+        Dependencies
+      </h2>
+      <p>
+        The terms <dfn data-cite="HTML#the-link-element">link</dfn>, 
```

Comment:
@marcoscaceres is this the preferred way? DFN the concept and use it throughout? It seems like in other examples we talked about `<a data-dice=...>` pattern instead?

I'm a little wary of maintaining a manual glossary.